pages/expense.js

```diff
@@ -313,7 +313,7 @@ class ExpensePage extends React.Component {
     const collective = expense?.account;
     const host = collective?.host;
     const hasAttachedFiles = expense?.attachedFiles?.length > 0;
-    const showTaxFormMsg = expense?.requiredLegalDocuments.includes('US_TAX_FORM') && expense?.permissions.canEdit;
+    const showTaxFormMsg = includes(expense?.requiredLegalDocuments, 'US_TAX_FORM');
```

Comment:
Cool, we're removing the permission check here because we're already checking if the user has permission in the resolver. :+1:

Reply:
Yep! I really embrace the idea that permissions checks should be centralized in the API